### ISSUE TITLE
 There is no pagination in this view

### ISSUE THREAD

**Issue by truthsword** [cypht-org/cypht#1045]
## 💬 Question
When selecting Everything, 7 messages appear, although the setting is for 4 weeks. Under the messages I see:

**There is no pagination in this view, please visit the individual mail boxes.**

What am I missing here? What do I do?

**Comment by marclaporte** [comment]
Lower on the left, you should see all your email accounts.

**Comment by truthsword** [comment]
Yes, but why aren't the full 20 messages populated in the Everything view.

**Comment by marclaporte** [comment]
Looks like a bug. Sorry about that. Let's fix it.

- Which version of Cypht?
- How was it installed (manual, Docker, YunoHost, part of Tiki)? It should not matter but it's best when devs replicate the environment.
- What is the time range of messages that are appearing OK vs missing?

Thanks!

**Comment by truthsword** [comment]
OK... Let’s start in small steps as I've redone server accounts and profiles. The combined folders seem off (I'll eventually get to Everything). Let's look at “Junk”. I have 3 accounts. It looks as though the Junk view is drawing from the drafts folder from each account.
![](https://i.imgur.com/txeM8HK.png)
I'm unsure how to change this. The Trash view also shows these same 3 draft folders, but not the trash folders.

Presently I have ~15 junk messages across the 3 accounts. But they do not appear as the resource for the Junk folder appears to be “drafts”. If I've caused this, I don't recognize where.

This is a docker container (“latest”).

**Comment by marclaporte** [comment]
> It looks as though the Junk view is drawing from the drafts folder from each account.

Yes, I have seen this before and it's fixed. I think by @josaphatim.

@truthsword: We want you on latest 2.0.x or master (which are almost identical at the moment. Until https://hub.docker.com/u/cypht is operational, can you use https://hub.docker.com/r/jonocodes/cypht to test https://github.com/cypht-org/cypht/pull/1001?

**Comment by marclaporte** [comment]
> This is a docker container (“latest”).

https://github.com/cypht-org/cypht/issues/1047

**Comment by truthsword** [comment]
OK… The new container is up using ` image: jonocodes/cypht:2.0.1-docker-wip` (docker compose)

I tried running it without the tag, but the screen went white often, and a browser refresh only returned me to the “home” screen. The tagged image seems to redraw as expected.

I'll add accounts/servers over the next day or so. Thanks!

**Comment by josaphatim** [comment]
Hello @truthsword. I just test using jonocodes/cypht:2.0.1-docker-wip image. I'm having 20 messages in everything page. But I recommend to use the official image. @Yannick243 fixed issues with combined Junk, Trash, Drafts. Thanks.

**Comment by marclaporte** [comment]
@truthsword Please test https://github.com/cypht-org/cypht/releases/tag/v2.2.0

Also available via an improved Docker:
https://hub.docker.com/r/cypht/cypht

**Comment by marclaporte** [comment]
https://github.com/cypht-org/cypht/releases/tag/v2.4.0 is now released

**Comment by marclaporte** [comment]
@truthsword The monthly Cypht community meetings are a great opportunity to show bugs: https://github.com/cypht-org/cypht/wiki/Monthly-Community-Meetings
